laba_5.cpp

```diff
@@ -0,0 +1,81 @@
+#include "student.h"
+#include "group.h"
+
+int main()
+{
+
+    int num_of_students;
+    cout << "how many students in group?\t";
+    cin >> num_of_students;
+    vector<Student> list_of_students;
+    for (int index = 0; index < num_of_students; index++)
+    {
+        list_of_students.push_back(Student());
+    }
+
+    for (int i = 0; i < num_of_students; i++)
+    {
+        int num_of_marks;
+        cout << "name of student: ";
+        cin >> list_of_students[i].name;
+        cout << "how many marks " << list_of_students[i].name << " have?\t" << endl;
+        cin >> num_of_marks;
+        list_of_students[i].marks_arr_create(num_of_marks);
+        list_of_students[i].average_of_marks_create();
+    }
+    cout << "____________________________________________________" << endl;
+    cout << "student name:\t"
+         << "student marks:\t"
+         << "mark average:\t" << endl;
+    cout << "____________________________________________________" << endl;
+
+    for (int i = 0; i < num_of_students; i++)
+    {
+        cout << list_of_students[i].name << " \t";
+        list_of_students[i].marks_output();
+        cout << list_of_students[i].average_of_marks << endl;
+    }
+
+    Group ir_12;
+
+    float arr_average_of_marks [num_of_students];
```

Comment:
'float arr_average_of_marks [num_of_students];'
return type is "float" but function aint return nothing